android/AndroidManifest.xml

```diff
@@ -4,6 +4,9 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:installLocation="auto">
 
+  <uses-sdk android:minSdkVersion="23" android:targetSdkVersion="32" />
```

Comment:
As I mentioned before, minsdk and targetsdk should not be modified here, they're taken from gradle.properties.

Reply:
Fixed